kornia/color/sepia.py

```diff
@@ -0,0 +1,81 @@
+import torch
+
+from kornia.core import Module, Tensor
+
+
+def sepia_from_rgb(input: Tensor, rescale: bool = True, eps: float = 1e-6) -> Tensor:
+    r"""Apply to a tensor the sepia filter.
+
+    Args:
+        input: the input tensor with shape of :math:`(*, C, H, W)`.
+        rescale: If True, the output tensor will be rescaled (max values be 1. or 255).
+        eps: scalar to enforce numerical stability.
+
+    Returns:
+        - Tensor: The sepia tensor of same size and numbers of channels
```

Comment:
` - Tensor:` -> 'out'. It uses typing from typing :)

Reply:
You mean, update to just: `Tensor: The sepia tensor of same size and numbers of channels`